.github/workflows/python-package.yml

```diff
@@ -16,12 +16,10 @@ jobs:
     strategy:
       matrix:
         python-version: ['3.10', '3.11', '3.12']
-        numpy_version: ['>=1.24.0', '==1.23.*']
+        numpy_version: ['>=2.1', '==1.24.*', '==1.26.*']
```

Comment:
I remain unconvinced that we need to test on 1.24 and 1.26 - I don't think there were major differences between the two, so testing on the oldest and newest version of our supported numpy should be good enough?

Reply:
You win! 😉